docs/installation_guide.md

```diff
@@ -121,8 +121,8 @@ _NOTE: For detailed information on storage backend of a registry, refer to [Regi
 #### Finishing installation and starting Harbor
 Once **harbor.cfg** and storage backend (optional) are configured, install and start Harbor using the ```install.sh``` script.  Note that it may take some time for the online installer to download Harbor images from Docker hub.  
 
-##### Default installation (without Notary)
-After version 1.1.0, Harbor has integrated with Notary, but by default the installation does not include Notary service.
+##### Default installation (without Notary/Clair)
+After version 1.1.0, Harbor has integrated with Notary and Clair, but by default the installation does not include Notary or Clair service.
```

Comment:
After version 1.1.0, Harbor has integrated with Notary and Clair,
==>
After version 1.1.0, Harbor has integrated with Notary and Clair (for vulnerability scanning),